extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/LDAPGuacamoleProperties.java

```diff
@@ -144,5 +144,8 @@ private LDAPGuacamoleProperties() {}
         public String getName() { return "ldap-encryption-method"; }
 
     };
-
+    public static final StringGuacamoleProperty ADDITIONAL_LDAP_SEARCH_FILTERS = new StringGuacamoleProperty(){
+    	  @Override
+          public String getName() { return "ldap-additional-search-filter"; }
```

Comment:
Again, please try to follow the style of the rest of the code. Consider this existing property from earlier in this file:

```
    /**
     * The base DN to search for Guacamole configurations.
     */
    public static final StringGuacamoleProperty LDAP_CONFIG_BASE_DN = new StringGuacamoleProperty() {

        @Override
        public String getName() { return "ldap-config-base-dn"; }

    };
```

There is a descriptive property name which is consistent between the Java code (`LDAP_CONFIG_BASE_DN`) and the property that would be used in guacamole.properties (`ldap-config-base-dn`), and the Java property is properly documented with a JavaDoc comment.

Compare to:

```
    public static final StringGuacamoleProperty ADDITIONAL_LDAP_SEARCH_FILTERS = new StringGuacamoleProperty(){
    	  @Override
          public String getName() { return "ldap-additional-search-filter"; }
    };
```

Note:

1. The JavaDoc comment is missing.
2. The Java naming (`ADDITIONAL_LDAP_SEARCH_FILTERS`) does not match that of the property (`ldap-additional-search-filter`).
3. The spacing surrounding the `getName()` implementation with respect to blank lines is inconsistent with the rest of the code in this file.